{"name": "Checkout API response within invalid value and get correct response and error message", "status": "passed", "steps": [{"name": "Login account.", "status": "passed", "start": 1677719932461, "stop": 1677719933737}, {"name": "Go to prime page.", "status": "passed", "start": 1677719933737, "stop": 1677719935237}, {"name": "Get prime key.", "status": "passed", "start": 1677719935237, "stop": 1677719935958}, {"name": "Send OrderAPI invalid value.", "status": "passed", "start": 1677719935958, "stop": 1677719936533}, {"name": "Verify Http_status_code is correct.", "status": "passed", "start": 1677719936533, "stop": 1677719936533}, {"name": "Verify error message is correct.", "status": "passed", "start": 1677719936533, "stop": 1677719936534}], "attachments": [{"name": "log", "source": "f3c70cb6-d248-414d-8ea3-343f01e980d9-attachment.txt", "type": "text/plain"}], "start": 1677719931940, "stop": 1677719936534, "uuid": "3c2621ad-617b-4988-8fd0-8647bdc00368", "historyId": "0b177ccbc4415c16fcfd6074b05d06ed", "testCaseId": "5822bd466a2acfebb7e50b60e25009b9", "fullName": "test_api.test_order#test_checkout_order_failure", "labels": [{"name": "feature", "value": "Checkout order API"}, {"name": "story", "value": "Checkout order failure."}, {"name": "tag", "value": "OrderAPI"}, {"name": "parentSuite", "value": "test_api"}, {"name": "suite", "value": "test_order"}, {"name": "host", "value": "Lins-MBP.GoMore-3F"}, {"name": "thread", "value": "42545-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_api.test_order"}]}